### ISSUE TITLE
Leaking memory

### ISSUE THREAD

**Issue by gjcarneiro** [MagicStack/uvloop#201]
* **uvloop version**: 0.11.1
* **Python version**: 3.7.0
* **Platform**: linux (docker)
* **Can you reproduce the bug with `PYTHONASYNCIODEBUG` in env?**: yes

Take the following program, which is silly, but to demonstrate that it doesn't leak memory, even with gc disabled:

```python3
import uvloop
import asyncio
import gc


async def main() -> None:
    while True:
        try:
            await asyncio.wait_for(asyncio.sleep(1), 0.001)
        except asyncio.TimeoutError:
            pass


if __name__ == "__main__":
    # asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
    gc.disable()
    asyncio.run(main())
```

Now if you uncomment the line to enable uvloop and run again you will observe that it starts to leak memory like crazy.

In fact, even with gc enabled, it leaks memory, so I guess the GC is a red herring.

**Comment by 1st1** [comment]
See the release notes for 0.11.2: https://github.com/MagicStack/uvloop/releases/tag/v0.11.2 :)

**Comment by gjcarneiro** [comment]
Ah, lovely!  Indeed it's fixed in v0.11.2!  Sorry.

Sorry, I  was convinced I was on the latest uvloop release, but clearly was confused.  Too many microservices...
